eosc/cmd/voteRecast.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+
+	eos "github.com/eoscanada/eos-go"
+	"github.com/eoscanada/eos-go/system"
+	"github.com/spf13/cobra"
+)
+
+var voteRecastCmd = &cobra.Command{
+	Use:   "recast [voter name]",
+	Short: "Recast your vote for the same producers",
+	Args:  cobra.MinimumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		api := getAPI()
+		voterName := toAccount(args[0], "voter name")
+
+		response, err := api.GetTableRows(
+			eos.GetTableRowsRequest{
+				Code:       "eosio",
+				Scope:      "eosio",
+				Table:      "voters",
+				JSON:       true,
+				LowerBound: string(voterName),
+				Limit:      1,
+			},
+		)
+		errorCheck("get table row", err)
+
+		var voterInfo []eos.VoterInfo
+		err = response.JSONToStructs(&voterInfo)
+		errorCheck("reading voter_info", err)
+
+		var found bool
+		var producerNames []eos.AccountName
+		for _, info := range voterInfo {
+			if info.Owner == voterName {
+				found = true
+				producerNames = info.Producers
+			}
+		}
+		if !found {
+			errorCheck("voter_info", fmt.Errorf("not found"))
```

Comment:
I feel we should improve the error message, something in the like of: "Not possible to recast vote since we did not found any producers you voted for.